Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -313,11 +324,17 @@ - (void)updateFooter
         !attributedText) {
         self.footerLabel.hidden = YES;
         [self.viewConstraints addObjectsFromArray:@[
+            [self.footerView autoPinEdge:ALEdgeTop toEdge:ALEdgeBottom ofView:self.messageBubbleView],
             [self.footerView autoSetDimension:ALDimensionHeight toSize:0],
         ]];
         return;
     }
 
+    [self.footerView autoPinEdge:ALEdgeTop
```

Comment:
Doe we need to capture this in the self.viewConstraints so as not to redundantly constrain?

Reply:
Good catch.